.github/workflows/ci.yml

```diff
@@ -1,70 +1,53 @@
 name: CI
 
-on:
-  push:
-  pull_request:
-    branches:
-      # Branches from forks have the form 'user:branch-name' so we only run
-      # this job on pull_request events for branches that look like fork
-      # branches. Without this we would end up running this job twice for non
-      # forked PRs, once for the push and then once for opening the PR.
-      - '**:**'
-      - master
+on: [push, pull_request]
+
+env:
+  USE_SLJIT_COMMIT: 19309dc09600fa422c16c5453adc721f6a6e9c81
```

Comment:
Maybe add a comment about what date this commit is from. Just from the hash, it's kind of impossible to know if this is a new or old version.